data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,103 @@ mission "Arachi Cartoonist 2"
+					goto att
+				`	"I'll join the defenders."`
+					goto def
+				`	"Hang on, if this is a folk tale, how come you're reenacting it as if any side could win?"`
+			`	He chitters, as if expecting the question. "Witnessed by a minstrel, the battle was. Travelling on fields like these, he was, before the fort, and the raider camp, he spotted in the distance. Hid himself in foliage close to the scene, he had, and writing and painting it as it happened, he was, but when reached its climax, the battle had, swept up by the wind, many flowers were, and block his vision they did. Once settled down the wind had, over, the battle was. Spring it was, so 'Spring Siege' became the story's title."`
+			`	Seeing you have more questions now, he continues. "Silly and nonsensical at times, the story is. Know that, we do, and that simply looking to entertain a crowd, the minstrel was, but with all forms of classic tales, is that not the case?" He chitters some more, and looks at the group up on the hill, and the other in the camp, before continuing. "Foreign as it might seem, participate in it, will you not? Perhaps like it, you might."`
+			choice
+				`	"Alright, I'll join the attackers."`
+					goto att
+				`	"Alright, I'll join the defenders."`
+			label def
+			`	He brings you to the small fort up the hill. Its gates open as those inside (mostly Kimek as expected, but also a couple Arachi and a lone Saryd) are still getting ready. A small "farm" sits at the middle, with plastic replicas serving as the crops. The walls of the fort are made of various tall logs, densely covered by tufts of vicious, reddish thorns nearly as long as your arm. Inside you can see that a higher level with battlements is accessible via staircases on either side of the courtyard; you also notice that the defenders are all wearing a green strip of cloth over their clothing. Another Kimek organizer comes to meet you two at the gate, wearing a rugged leather vest that you assume is the Kimek equivalent of medieval armor. He greets his colleague, who introduces you and tells him you chose to join the defenders, before heading back down the hill.`
+			`	"Glad to have you as part of the garrison, our fort is," the armored Kimek begins. "Told about the enemy's objective, I assume you have been. Protect our farm, we must, so allow the produce to be stolen, we cannot." He tells you to follow him to the 'keep,' a shack too small for any non-Kimek, and gives you a long piece of green cloth. He then informs you that it is meant to be worn, so that others can tell what team you are on. As you wrap it around you like a scarf, you notice that, behind the shack, there is a smaller gate that grants entry to the fort. Once you're done, he gives you a brief tour of the rest of the fort, taking you up the battlements and explaining that, above the main gate, they have buckets of a water-like, slippery liquid. During the re-enactment, they are poured downhill and used to slow the attackers.`
```

Comment:
```suggestion
			`	"Glad to have you as part of the garrison, our fort is," the armored Kimek begins. "Told about the enemy's objective, I assume you have been. Protect our farm, we must, so allow the produce to be stolen, we cannot." He tells you to follow him to the "keep," a shack too small for any non-Kimek, and gives you a long piece of green cloth. He then informs you that it is meant to be worn, so that others can tell what team you are on. As you wrap it around you like a scarf, you notice that, behind the shack, there is a smaller gate that grants entry to the fort. Once you're done, he gives you a brief tour of the rest of the fort, taking you up the battlements and explaining that, above the main gate, they have buckets of a water-like, slippery liquid. During the re-enactment, they are poured downhill and used to slow the attackers.`
```